image-index.md

```diff
@@ -112,5 +117,7 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 }
 ```
 
+[cpufeatures.h]: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/x86/include/asm/cpufeatures.h
```

Comment:
For L108 (GitHub doesn't allow me to directly comment there)

* os.features -> features
* sse4 -> sse4_2 (please grep and update other files as well)